packages/Popover/src/Popover.js

```diff
@@ -89,7 +90,7 @@ const defaultProps = {
   getPositioningElement: null,
   getScrollContainer: null,
   shouldKeepFocus: false,
-  zIndex: 1,
+  zIndex: zValue(1),
```

Comment:
I would like to revisit all of the `default` values for `zIndex` props and align to a seven-layer model (for example the ListBox.Popover has a default of an arbitrary `100`). 

However, some thought needs to go into that, so for now I am not introducing any changes.